How-to-reset-qBittorrent-webui-password.md

```diff
@@ -0,0 +1,48 @@
+# How to reset qBittorrent webui password
```

Comment:
webui -> WebUI

Reply:
`How to reset qBittorrent webui password` -> `How to reset qBittorrent WebUI password`